README.md

```diff
@@ -2,10 +2,56 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+It is the mission of the Node.js Foundation and the TSC to provide a home for 
+openly-governed and community-managed open source projects that are focused on 
+supporting and providing benefit to the entire Node.js ecosystem.
+
+Project and Working Groups under the TSC are governed by the contributors
+and users of those projects.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
 For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
 
 If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
+## Projects, Working Groups and the Incubator
+
+The Node.js Foundation is structured to serve as a home for multiple 
+autonomous projects. Projects are first admitted into an incubation 
+phase where they are mentored until the project governance and 
+contributor base reaches a state of maturity. The TSC then works 
+with the project to establish an official charter and a seat on the TSC.
+
+The TSC also forms Working Groups to tackle tasks that don't scope to a single project. These 
+Working Groups can be given a seat on the TSC at the TSC's discretion.
+
+The goal of the Incubator is to:
+
+* Support the ecosystem of Node.js users and contributors by providing a 
+  home for openly governed and community managed projects built on, or 
+  built to support Node.js.
+* Mentor projects within the Node.js community on open governance and liberal 
+  contribution policies.
+* Ensure that the Node.js Foundation does not charter projects that do not 
+  adhere to the values upon which the Node.js Foundation was established. 
+  Projects must be: Transparent, Participatory and Effective.
+* Ensure that the Node.js Foundation and the TSC does not "pick a winner" in 
```

Comment:
Agreed, but we can also message this when we announce projects entering the foundation. The important thing is that people entering the incubator and the TSC understand this commitment so that we can speak to it consistently.